packages/drivers/driver-base/src/documentDeltaConnection.ts

```diff
@@ -248,7 +248,7 @@ export class DocumentDeltaConnection
 		return this.details.serviceConfiguration;
 	}
 
-	private checkNotDisposed() {
+	protected checkNotDisposed() {
```

Comment:
Nit: This method does more than just "check", it "asserts" that it isn't disposed. Since we're effectively exposing this as a new API (it was private before), this might be a good opportunity to rename it to something like "assertNotDisposed".

Reply:
Will do this work in a separate PR -- this one will just include new test